Sources/Apodini/Modifier/ResponseModifer.swift

```diff
@@ -130,7 +130,7 @@ extension Handler {
     public func response<T: ResponseTransformer>(
         _ responseTransformer: @escaping @autoclosure () -> (T)
     ) -> ResponseModifier<Self, T> where Self.Response == T.Response {
-        if Self.Response.self is ApodiniEncodable.Type {
+        if Self.Response.self is EncodableContainer.Type {
             preconditionFailure("Actions cannot be transformed directly. Use a transformer on the type that is wrapped by the Action instead.")
```

Comment:
There's several places in the code which operate on the assumption "if it conforms to `EncodableContainer` it must be an `Action`".
Is this the case, or am I misunderstanding something?
If yes, do we enforce this anywhere?
What about other types which might conform to `EncodableContainer`?

I think if `Action` really is the only type conforming to this protocol, and the protocol it's being used to determine whether an object is an Action (and as a type-erased way to call the action's accept method), it might make more sense to rename it to `ActionProtocol` or something like that.